genomic-data-product.md

```diff
@@ -1,152 +1,163 @@
-### **Genome annotations**
-
-
+### Genome annotations
```

Comment:
Suggest removing

Reply:
done in https://github.com/terraref/documentation/pull/167/commits/4f5944260f61cf261d3aa144dc9297d961995dbf